src/core/exception.d

```diff
@@ -24,6 +24,8 @@ private
     {
         assertHandlerType assertHandler   = null;
     }
+
+    extern (C) int  snprintf(char* s, size_t n, in char* format, ...);
```

Comment:
I've tried not to import anything, keeping the module clean. I will do it, though.